src/Menu/Provider/GroupMenuProvider.php

```diff
@@ -85,11 +85,22 @@ public function __construct(FactoryInterface $menuFactory, Pool $pool, $checker
      */
     public function get($name, array $options = [])
     {
+        /**
+         * @var array{
+         *             label: string,
+         *             label_catalogue: string,
+         *             icon: string,
+         *             on_top?: bool,
+         *             keep_open: bool,
+         *             provider: string,
+         *             items: list,
+         *             } $group
```

Comment:
php-cs-fixer doing his things with the indentation, can you check if there is a rule to fix this indentations? maybe another @sonata-project/contributors knows how to fix this

Reply:
I always saw this syntax on a single line

Reply:
@VincentLanglet @jordisala1991 

okay, I found something

If I do, `@var-phpstan` it does not align anything.
BUT, my PHPStorm array keys (of `$group` variable) are not `CTRL+click`ing anymore with new tag (before that I can click on `$group['on_top']`, the key `on_top` and go to declaration of PhpDoc of the variable)

What the best way to solve this?

Reply:
Can you inline the declaration? it will work for now

Reply:
sure